www/site/content/docs/reference/config.md

```diff
@@ -0,0 +1,263 @@
++++
+title = "Chef Workstation Configuration"
+linkTitle = "Chef Workstation Configuration"
+[menu]
+  [menu.docs]
+    parent = "Reference"
+    weight = "10"
++++
+
+# Configuration
+
+Chef Workstation App and `chef-run` configuration is managed in `config.toml`.
+Chef Workstation will create a `config.toml` the first time you use `chef-run`, if one does not already exist.
+To configure of other tools, see their respective pages found in the [ChefDK documentation](https://docs.chef.io/about_chefdk.html).
+
+## Default location
+
+Chef Workstation looks for the `config.toml` in a default location.
+
+### Windows
+
+* Powershell: `$env:USERPROFILE\.chef-workstation\config.toml`
+* cmd.exe: `%USERPROFILE%\.chef-workstation\config.toml`
+
+### Linux and Mac
+
+`/home/$USER/.chef-workstation/config.toml`
+
+## Settings
+
+### [telemetry]
+
+Configure telemetry behaviors for Chef Workstation components.
+
+#### Example
+
+```toml
+[telemetry]
+enable=true
+dev=false
+```
+
+#### Option: enable
+
+* Description: When `true`, anonymous usage data and bug reports are sent to Chef.
+* Default: `true`
+* Valid values: `true`, `false`
+* Environment:
+* `CHEF_TELEMETRY_OPT_OUT`: when set to any value, `chef-run` will not capture or send telemetry data.* Notes:
+* See Chef's [Privacy Statement](https://www.chef.sh/docs/chef-workstation/privacy/) for the type and usage of gathered data.* Used by: `chef-run`
+
+#### Option: dev
```

Comment:
Do we even want to include the `dev` field in the config documentation? I would think not

Reply:
Nope, we can nuke it